aggregator/views.py

```diff
@@ -119,5 +121,27 @@ class LocalDjangoCommunitiesListView(ListView):
     context_object_name = "django_communities"
     template_name = "aggregator/local-django-community.html"
 
+    def get_queryset(self):
+        return self.model.objects.filter(approval_status=APPROVED_FEED).order_by("continent")
+
+
+class LocalDjangoCommunitiesAddView(CreateView):
+    """
+    Shows a list of community meetups
+    """
+
+    model = LocalDjangoCommunity
+    form_class = LocalDjangoCommunityModelForm
+    context_object_name = "django_communities"
+    template_name = "aggregator/edit-local-django-community.html"
+
+    def get_success_url(self):
+        return reverse("local-django-communities")
+
     def get_queryset(self):
         return self.model.objects.all().order_by("continent").values()
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["adding"] = True
+        return context
```

Comment:
This view is public so someone can spam our database with millions new communities. I would at least add here some throttling to prevent that. Maybe we can think about giving access here only to some verified users like for adding Feeds?